robot_modeling/DQ_SerialManipulator.m

```diff
@@ -13,15 +13,30 @@
 % The first row of 'A' contains the joint offsets. More specifically,
 % theta_i is the offset for the i-th joint.
 %
-% DQ_SerialManipulator Methods (Concrete):
+% DQ_SerialManipulator Methods (Abstract):
 %       get_dim_configuration_space - Return the dimension of the configuration space.
 %       fkm - Compute the forward kinematics while taking into account base and end-effector's rigid transformations.
 %       plot - Plots the serial manipulator.
 %       pose_jacobian - Compute the pose Jacobian while taking into account base's and end-effector's rigid transformations.
 %       pose_jacobian_derivative - Compute the time derivative of the pose Jacobian.
 %       raw_fkm - Compute the FKM without taking into account base's and end-effector's rigid transformations.
 %       raw_pose_jacobian - Compute the pose Jacobian without taking into account base's and end-effector's rigid transformations.
+
+%
+% DQ_SerialManipulator Methods (Concrete):
+%       get_effector - Get the end-effector rigid transformation with respect to the last frame in the kinematic chain.
 %       set_effector - Set an arbitrary end-effector rigid transformation with respect to the last frame in the kinematic chain.
+%       get_lower_q_limit - Returns the lower bound of the robot configuration.
+%       set_lower_q_limit - Sets the lower bound of the robot configuration.
+%       get_lower_q_dot_limit () - Returns the lower bound of the robot configuration velocity.
```

Comment:
Some methods have brackets and some don't. Is there a specific reason for this or perhaps just a typo?